plugins/holland.backup.mariabackup/holland/backup/mariabackup/util.py

```diff
@@ -116,7 +116,7 @@ def apply_mariabackup_logfile(mb_cfg, backupdir):
     innobackupex = mb_cfg["innobackupex"]
     if not isabs(innobackupex):
         innobackupex = which(innobackupex)
-    args = [innobackupex, "--prepare", backupdir]
+    args = [innobackupex, "--prepare", "--target-dir=" + backupdir + "/data"]
```

Comment:
We should probably use `os.path.join` here just to be safe; e.g `args = [innobackupex, "--prepare", "--target-dir=" + path.join(backupdir, "/data")]`